src/scripts/colours.js

```diff
@@ -276,8 +276,10 @@ class NewTab extends Component {
         this.state.colour
     };
 
+    // Text styles
+    let textClass = Colours.isDark(...(Colours.hexToRgb(this.state.colour.substring(1))))?"white":"dark-grey"
```

Comment:
A more generic name than `textClass` would be preferred since this will be used for icons in the future as well. Same goes the `white`/`dark-grey`.